frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
```

Comment:
use import of common component only from `libs/components`

Reply:
Was not resolved